docs/setup/ubuntu.md

```diff
@@ -0,0 +1,102 @@
+# Ubuntu Setup
+
+This guide details the steps to set up Huxley on Ubuntu. Use this if you'd like to customize your setup, or had trouble with the setup script.
+
+### Pip
+Pip allows you to install python packages that will help you develop python applications.
+	
+	$ sudo apt-get update 
+	# updates your repository so you can get the newest version of the package
+
+	$ sudo apt-get install python-pip
+
+### Virtualenv
+Virtualenv separates your project's Python modules from the rest of your system to prevent conflicts. We'll install virtualenv and a useful utility built on top of it, virtualenvwrapper.
+
+    $ sudo pip install virtualenv
+    $ sudo pip install virtualenvwrapper
+	$ mkdir ~/.virtualenvs
```

Comment:
Seems like some weird indentation stuff going on here. Make sure your editor is configured to use spaces as tabs.